features/step_definitions/_then_steps.rb

```diff
@@ -263,8 +263,8 @@
   task_div[:class].should_not include("error")
 end
 
-Then /^I should be notified that the work_package "(.+?)" is an invalid parent to the work_package "(.+?)" because of cross project limitations$/ do |parent_name, child_name|
-  step %Q{I should see "Parent is invalid because the work_package '#{child_name}' is a backlogs task and as such can not have the backlogs story '#{parent_name}' as it's parent as long as the story is in a different project" within "#errorExplanation"}
+Then /^I should be notified that the work_package "(.+?)" is an invalid parent to the work package "(.+?)" because of cross project limitations$/ do |parent_name, child_name|
```

Comment:
Unused block argument - `parent_name`. If it's necessary, use `_` or `_parent_name` as an argument name to indicate that it won't be used.<br>Line is too long. [174/100]